code-challenges.test.js

```diff
@@ -52,5 +117,36 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+/**
+ * - If the array is empty, return an empty array
+ * - Loop the entire array
+ * - Add the accumulative sum to a variable called "sumList"
+ * - Return sumList
+ */
+
+const accumulatedSum = (arr) => {
+  if (arr.length === 0) return [];
+
+  const sumList = [];
+  let j = 0;
+
+  for (let i = 0; i < arr.length; i++) {
+    j += arr[i]
+    sumList.push(j);
+  }
+  return sumList;
+};
 
 // b) Create the function that makes the test pass.
+
+describe('accumulatedSum()', () => {
+  it('should return [2, 4, 45, 9] when given fullArr1', () => {
+    expect(accumulatedSum(numbersToAdd1)).toEqual([2, 6, 51, 60]);
+  });
+  it('should return [0, 7, -8, 12] when given fullArr2', () => {
+   expect(accumulatedSum(numbersToAdd2)).toEqual([0, 7, -1, 11]);
+  });
+  it('should return [] when given fullArr3', () => {
+   expect(accumulatedSum(numbersToAdd3)).toEqual([]);
+ });
+});
```

Comment:
Your code looks good. I see you are writing your tests after your functions. In the future try writing your tests first, seeing them fail, then writing the code to see them pass.